tomls/omnibus-base-goerli-andromeda/perps/xrp.toml

```diff
@@ -12,10 +12,10 @@ defaultValue = "1000000"
 defaultValue = "9"
 
 [setting.perpsXrpMakerFeeRatio]
-defaultValue = "0.0007"
+defaultValue = "0.0003"
```

Comment:
in andromeda branch we also have fixed `pythXrpFeedId` (and for LTC), which are the main culprit.

Reply:
How did that affect maker/taker fees

Reply:
It does not, but if you change copypaste bugs in xrp config you might as well fix the feed id too.